packages/appstore/src/assets/svgs/wallets/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import UpgradeDesktop from 'Assets/svgs/wallets/wallets-upgrade-desktop.svg';
+import UpgradeMobile from 'Assets/svgs/wallets/wallets-upgrade-mobile.svg';
+
+export const WalletsImagesList = {
+    UpgradeDesktop,
+    UpgradeMobile,
+};
+
+type WalletsImageProps<T> = {
+    banner: T;
+    className?: string;
+};
+
+const WalletsImage = ({ banner, className }: WalletsImageProps<keyof typeof WalletsImagesList>) => {
+    const Component = WalletsImagesList[banner] as React.ElementType;
+
+    return <Component className={className} data-testid={banner} />;
```

Comment:
Please add more props like size, icon to make a more general component which is usable for the rest of the icons for this wallets project.